Server/AuthService/schemas.py

```
@@ -0,0 +1,222 @@
+        from_attributes = True
+        json_schema_extra = {
+            "example": {
+                "id": 1,
+                "email": "admin@faceofmind.it.com",
+                "first_name": "Admin",
+                "last_name": "User",
+                "role": "admin",
+                "status": "active",
+                "created_at": "2024-01-01T00:00:00Z"
+            }
+        }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)